### ISSUE TITLE
Automatic .strm creation not working (Tested on the "Shared with Me" Files)

### ISSUE THREAD

**Issue by PaulinoRBJ** [ddurdle/GDrive-for-KODI#175]
I get files shared to me that I expect GDrive to create a .strm to, that they're indexed into my library. I enabled the automatic creation option and also tried the scheduling, none seem to work.

**Comment by 4luk** [comment]
Yeah mine won't add new movie added and won't update files that are rename/deleted. Did you find a way to fix this? Thanks
